caas/kubernetes/provider/resources/events.go

```diff
@@ -15,20 +15,31 @@ import (
 
 const maxEventsToPage = 100
 
+func listEventsForObject(
```

Comment:
We should just export this one and delete the one below. Having 2 methods with the same name is confusing.
From what I can see the place where we call the one below
```
return resources.ListEventsForObject(context.TODO(), objName, objKind, k.client().CoreV1().Events(k.namespace))
```
can be rewritten as
```
return resources.ListEventsForObject(context.TODO(), k.client(), k.namespace, objName, objKind)
```